docs/flows.md

```diff
@@ -23,16 +28,16 @@ Input and output schemas for flows can be defined using `zod`.
 import { defineFlow } from '@genkit-ai/flow';
 import * as z from 'zod';
 
-export const myFlow = defineFlow(
+export const menuSuggestionFlow = defineFlow(
```

Comment:
I'm not sure this example _needs_ to change from the original

Reply:
Do you think I should revert it back? I figured we would just use the menu suggestion theme throughout.